frontend/vite.config.ts

```diff
@@ -49,6 +49,18 @@ export default mergeConfig(
                     target: UNLEASH_API,
                     changeOrigin: true,
                 },
+                [`${UNLEASH_BASE_PATH}health`]: {
+                    target: UNLEASH_API,
+                    changeOrigin: true,
+                },
+                [`${UNLEASH_BASE_PATH}invite`]: {
```

Comment:
Discussion point: should all authenticated endpoints be not in `/api`? How is `/api/frontend` authenticated with other type of tokens?

Reply:
/api is covered by middleware, so `/api/frontend` is going through the `api-token-middleware`